extensions/ql-vscode/src/query-serialization.ts

```diff
@@ -54,7 +54,7 @@ export async function slurpQueryHistory(fsPath: string): Promise<QueryHistoryInf
     // most likely another workspace has deleted them because the
     // queries aged out.
     return asyncFilter(parsedQueries, async (q) => {
-      if (q.t === 'remote') {
+      if (q.t === 'remote' || q.t === 'variant-analysis') {
```

Comment:
On line 18, there is a `version` property that gets added. We probably want to bump this to version 2. This would mean that in the rare case someone downgrades from a version of the extension with the `variant-analysis` type to one without, they won't have errors when loading query history (it _will_ mean that they lose their query history, but I think that is a reasonable tradeoff).